module3-crud-and-data-models/r1.1-principles-of-setting-up-your-schema/README.md

```diff
@@ -0,0 +1,50 @@
+## Principles of setting up your schema
+
+Poorly designed databases can cause many problems, including a waste of resources, difficult maintenance, and faulty performance. That's why having a great database schema design is a crucial part of effective data management.
```

Comment:
including wasting resources, making maintenance difficult, and hindering performance